lib/rules/declaration-property-value-no-unknown/index.mjs

```diff
@@ -202,15 +202,15 @@ const rule = (primary, secondaryOptions) => {
 /**
  * @see csstree/csstree#164 min, max, clamp
  * @see csstree/csstree#245 env
+ * @todo create csstree issue for shape and calc-size
  * @param {import('css-tree').CssNode} cssTreeNode
  * @returns {boolean}
  */
 function containsUnsupportedFunction(cssTreeNode) {
+	const functions = ['clamp', 'min', 'max', 'env', 'shape', 'calc-size'];
```

Comment:
Can you omit `shape` both from the implementation and the titles?

This was not part of the original bug report and isn't actually the same scenario.

`shape()` is an extremely new spec and isn't a math function. If you add the syntax definitions validation also works just fine.

Reply:
> If you add the syntax definitions validation also works just fine.

That's important.

> Can you omit `shape` both from the implementation and the titles?

Will do.